### DISCUSSION TITLE
Sensor battery_charge is zero after upgrading to v1.10.2b3

### DISCUSSION THREAD

**Discussion by abuesen** [nathanmarlor/foxess_modbus#578] [Support] [answered]
I've updated two days ago to test the version-number on H3. It works. But now the `battery_charge` is `0` permanently while `battery_discharge` is not plausible not more than 500W.

Unfortunaly I can not prove the situation began the same time I updated (don't know where to see the time of the update).
HACS denies to revert to v1.10.1 I don't know why. Maybe one has to use the update-service, now.

I tried to investigate but didn't find the point.  With `modpoll -m tcp -a 3 -1 -0 -r 31036 192.168.x.x` it shows the right power like `-4661`. Today the battery charged with 4-7kW. This is missing in the history as you see in the 

![grafik](https://github.com/nathanmarlor/foxess_modbus/assets/136979340/b68bcabb-dfde-4274-af15-dd4dbe48d40c)

![grafik](https://github.com/nathanmarlor/foxess_modbus/assets/136979340/988d6ae3-aa17-4b7f-9c40-7909bfb7362d)

**Comment by abuesen** [comment]
The logs do not show update of `31036`.
[home-assistant_foxess_modbus_2024-03-24T11-37-06.885Z.log](https://github.com/nathanmarlor/foxess_modbus/files/14735397/home-assistant_foxess_modbus_2024-03-24T11-37-06.885Z.log)

**Comment by canton7** [comment, marked as answer]
Thanks! Duplicate of #573
